CHANGES

```diff
@@ -1,4 +1,4 @@
-2.0~beta
+2.0~alpha4
```

Comment:
Yeah, I saw the `beta` update and though "Oh, I haven't updated since a few weeks, that's a good occasion!"